prac_08/silver_service_taxi_test.py

```diff
@@ -0,0 +1,10 @@
+from prac_08.silver_service_taxi import SilverServiceTaxi
+
+silver_service_taxi = SilverServiceTaxi(200, "Hummer", 2)
+print(silver_service_taxi)
+silver_service_taxi.drive(18)
+print(silver_service_taxi)
+
+print(silver_service_taxi.get_fare())
+test = float(silver_service_taxi.get_silver_fare())
```

Comment:
Why do you need to convert a float to a float? Or doesn't the method return a float...? it should.